daemon/network.go

```diff
@@ -96,15 +96,7 @@ func (daemon *Daemon) GetNetworksByID(partialID string) []libnetwork.Network {
 
 // getAllNetworks returns a list containing all networks
 func (daemon *Daemon) getAllNetworks() []libnetwork.Network {
-	c := daemon.netController
-	list := []libnetwork.Network{}
```

Comment:
I am afraid that in libnetwork/controller.go: https://github.com/docker/libnetwork/blob/master/controller.go#L842, the list is a nil, while the removed part we found it is an empty slice.
And we know that by default, there will be several networks in store, but I am not sure if it will affects the API.